docs/installation-and-operations/changing-to-bim-edition/README.md

```diff
@@ -77,3 +77,33 @@ OpenProject installations with a valid Enterprise Edition token can switch to th
 theme.
 
 Go to _-> Administration -> Design_ and from the _Themes_ drop down menu choose _OpenProject BIM_.
+
+### Docker
+
+##### OpenProject BIM Edition with Docker Compose
```

Comment:
As mentioned below: the markdown headline hierarchy jumps from 3 hashes to 5, missing out 4. That looks semantically wrong to me.

Reply:
Question: who is a normal docker administrator? i suppose we do not have any in OP because they are all involved as developers too

Reply:
done